integration/common.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2022 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package integration
+
+import (
+	fut "github.com/eclipse-kanto/file-upload/integration"
+)
+
+const (
+	featureID = "BackupAndRestore"
```

Comment:
This constant + the operation constants could be seen outside of the integration tests as well. Does it make sense for them to be available for the whole project in one place?

Reply:
Maybe it makes sense to use a common constants for feature ID, as well as _backup_ and _restore_ operations.